README.md

```diff
@@ -1 +1,24 @@
-# tilt-extensions
\ No newline at end of file
+# Tilt Extensions
```

Comment:
I suspect it'd make sense to make this page more about browsing / using extensions than authoring them. I wouldn't be surprised if page loads with intent to consume/browse extensions outnumber page loads with intent to contribute > 20:1 - maybe it'd make sense to put the contributing stuff in a section at the end, or in a separate doc?

It's probably not very important either way for when we're just showing it to people at the start, so I guess it doesn't need to block.

Reply:
+1 prioritize browsing over contributing

Reply:
I suggest adding a link to some document that explains how to *use* an extension.